requirements-travis.txt

```diff
@@ -6,4 +6,5 @@ aexpect==1.6.4
 simplejson==3.8.0
 netaddr==0.7.19
 netifaces==0.11.0
+pycodestyle==2.8.0
```

Comment:
Is there a chance to either fix the E275 issues or temporarily disable the corresponding check and fix that later , instead?

Reply:
@luckyh 
We can find detailed error in any failed PR, for example: https://github.com/avocado-framework/avocado-vt/runs/7603849841?check_suite_focus=true
The short summary for this follow as:
1)Mainly it consists of two categories
2)The first category is E275 missing whitespace after keyword. This is easy part to be fixed
3)The second category is I1101:3460,14: get_ip_address_by_interface: Module 'netifaces' has no 'AF_INET6' member, but source is unavailable. I assume this is previously existed issue, which we not resolve before

Reply:
> 3)The second category is I1101:3460,14: get_ip_address_by_interface: Module 'netifaces' has no 'AF_INET6' member, but source is unavailable. I assume this is previously existed issue, which we not resolve before

Actually these ones are merely warnings, and you could even see that in a passed check, for example https://github.com/avocado-framework/avocado-vt/runs/7335639521?check_suite_focus=true